### PR TITLE
Session02

### PR DESCRIPTION
**Pull request by michaeluduhiri**
These are the files for the session 2 exercise

### CODE REVIEW

File: students/MichaelUduhiri/Session02/series.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+
+def fibonacci(n):
+    """ compute the nth Fibonacci number """
+    x,y = 1,1
+    for num in range(n-1):
+        x,y = y,x+y
+    return x
+    print (fibonacci(0))
+    pass
```
Comment (charlesrobison) [inline students/MichaelUduhiri/Session02/series.py]: pass not needed here

Comment (charlesrobison) [reply students/MichaelUduhiri/Session02/series.py]: Note fibanacci(0) should return zero.

File: students/MichaelUduhiri/Session02/series.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+    :param n0=0: value of zeroth element in the series
+    :param n1=1: value of first element in the series
+    
+    This function should generalize the fibonacci() and the lucas(),
+    so that this function works for any first two numbers for a sum series.
+    Once generalized that way, sum_series(n, 0, 1) should be equivalent to fibonacci(n).
+    And sum_series(n, 2, 1) should be equivalent to lucas(n).
+    """
+    pass
```
Comment (charlesrobison) [inline students/MichaelUduhiri/Session02/series.py]: This function doesn't appear complete.

**Comment by PythonCHB** [comment]
We haven't seen anything new here -- having trouble with git?

If you do a new push, it will show up in this PR.

Note: the pace is picking up! we don't want to you to get behind.

**Comment by PythonCHB** [comment]
yo udon't want to add files like ".Session02.swp" to git.

a
`git rm .Session02.swp`

should remove it.

**Comment by PythonCHB** [comment]
also not sure what `Session02.py` is -- empty?

`git rm Session02.py`

File: students/MichaelUduhiri/Session02/fizzbuzz.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+'''Write a program that prints the numbers from 1 to 100 inclusive.
```
Comment (PythonCHB) [inline students/MichaelUduhiri/Session02/fizzbuzz.py]: fizzbuzz looks good -- giving you credit for this.

File: students/MichaelUduhiri/Session02/series.py (hunk 1 above)
Comment (PythonCHB) [reply students/MichaelUduhiri/Session02/series.py]: and finish this...

Comment (michaeluduhiri) [reply students/MichaelUduhiri/Session02/series.py]: Alright. Will take another look.

Comment (michaeluduhiri) [reply students/MichaelUduhiri/Session02/series.py]: I may have overwritten what I had for grid printer during a previous save request. Will try again

**Comment by michaeluduhiri** [comment]
> We haven't seen anything new here -- having trouble with git?
> 
> If you do a new push, it will show up in this PR.
> 
> Note: the pace is picking up! we don't want to you to get behind.

File: students/MichaelUduhiri/Session02/fizzbuzz.py (hunk 3 above)
Comment (michaeluduhiri) [reply students/MichaelUduhiri/Session02/fizzbuzz.py]: thanks

**Comment by PythonCHB** [comment]
Idon't want to merge this 'till you remove:

.Session02.swp

that should be a simple:
```
git rm .Session02.swp
```
Then:

git commit -m

git push

File: students/MichaelUduhiri/Session03/slicing.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+def first_last_exchange(seq):
+    list_string = list(seq)
+    list_string[0], list_string[-1] = list_string[-1], list_string[0]
+    return "".join(list_string)
```
Comment (charlesrobison) [inline students/MichaelUduhiri/Session03/slicing.py]: This should be done with slicing - can you give it another try?

**Pull request merged**